TestingLowerBounds/ForMathlib/IntegralCongr2.lean

```diff
@@ -0,0 +1,30 @@
+import Mathlib.Probability.Kernel.Basic
+
+open MeasureTheory ProbabilityTheory
+
+namespace MeasureTheory
+
+--TODO: put this lemma in a separate file, then PR it to mathlib, I'm not sure it can just go in the same file as integral_congr_ae, since it uses the kernels. maybe we could do a simpler version with 2 probability measures instead of kernels. decide what to do with the 2 vertions, are they both useful?
+--I could have proven the second one using the first, but it is probabily easier to do them separately, also in this way we can put them in separate files without worring about dependencies
+--also about the names, if we put the two lemmas under different namespaces (the first one could go under something that contains kernel) we can give them the same name
+variable {α β: Type*} {mα : MeasurableSpace α} {mβ : MeasurableSpace β}
+variable {μ : Measure α} {ν : Measure β} {κ : kernel α β}
```

Comment:
You don't need to write `variable` at each new line. You can also simply indent the next line.